split_words/split_words.py

```diff
@@ -0,0 +1,37 @@
+from justifytext import justify
+
+with open('/Users/afoix/dango-git/python-challenge/split_words/text.txt') as text:
+    string = text.read()
+    split_words = string.split(' ')
+
+    # One where all the text is spaced to 40 characters
+    bugdet_user = int(input("Tell me a budget: "))
+    budget_char = bugdet_user
+    words_line = []
+    for word in split_words:
+        if len(word) >= budget_char:
+            words_line_space = ' '.join(words_line)
+            words_line_justified = justify(words_line_space, bugdet_user, justify_last_line = True)
+            print(' '.join(words_line_justified), end="\n")
```

Comment:
If you called `justify`, didn't it already combine all the text into a string? I think that means at this point you don't need to do the `join` call, you could just do `print(words_line_justified, end="\n")`.

Reply:
(What you have currently is working because `.join()` with something that is just a single string just returns the single string, but it could be simpler).

Reply:
Actually if I've remove what you said breaks the program so... I'm not sure this is correct

Reply:
interesting... maybe I am wrong about what `justify` returns then. No worries in that case :)